src/components/previewers/ImagePreviewer.jsx

```
@@ -25,9 +31,10 @@ const ImagePreviewer = ({
   userImage,
   userName,
   isEmoji = false,
+  title,
```

ESLint

[error] 'title' is missing in props validation

(react/prop-types)